### ISSUE TITLE
Error when making an rgb fits image. 

### ISSUE THREAD

**Issue by maitraiyeetiwari** [aplpy/aplpy#493]
I am trying to make an rgb cube using: 

import aplpy
aplpy.make_rgb_cube(['oi_63_mom0_0_8.fits','oi_63_mom0_8_20.fits','oi_63_mom0_8_20.fits'],'m8-oi-rgb.fits')

but get an error:
TypeError: concatenate() got an unexpected keyword argument 'dtype'. 

I was able to do this earlier without any problems and now it doesn't work.

**Comment by coude** [comment]
Did you update any of the packages in your working python environment since the last time this worked?

**Comment by maitraiyeetiwari** [comment]
Yes, I did upgrade numpy and numdifftools to the latest versions: 1.26.3 and  0.9.41, respectively.

**Comment by coude** [comment]
Assuming there is nothing wrong with the input FITS files, a compatibility issue between packages would be a plausible explanation.  I would suggest creating a new Python environment from scratch and then only installing aplpy (either with conda or pip), which should take care of all the dependencies you need to test make_rgb_cube. 

Out of curiosity, I successfully ran the same function using a bunch of (now published) HAWC+ image fits files for NGC 2024. It's essentially the same function you used, but this could serve as a check that it works on your side. Here is the [Github link](https://github.com/coude/RGB_Cube_Test).

Hopefully this helps a bit!

**Comment by coude** [comment]
Happy to help, I'm glad you found a workaround! 

If you have time at some point, it might be a good idea to recreate your python environment from scratch (or a new virtual environment), just in case it would be enough to fix those potential compatibility issues in the future.
